recipe/meta.yaml

```diff
@@ -70,8 +66,6 @@ requirements:
     - libtiff
     - libpng
     - openblas                       # [not win]
-    - blas 1.1 {{ blas_variant }}    # [not win]
-    - ffmpeg 4.1.*                   # [not win]
```

Comment:
master recipe still has ffmpeg 4.1.* in run section

Reply:
oops, wrong line deleted.